lib/crawly/worker.ex

```diff
@@ -64,6 +64,12 @@ defmodule Crawly.Worker do
     {:noreply, %{state | backoff: new_backoff}}
   end
 
+  # Catch ssl closed error coming from HTTPoison
+  def handle_info({:ssl_closed, _}, %{spider_name: _, backoff: backoff} = state) do
+    Crawly.Utils.send_after(self(), :work, backoff)
```

Comment:
It would be quite interesting to understand what's going on here & how to handle the error. I would also try to log something to the debug log.